.vsts-ci.yml

```diff
@@ -57,7 +57,7 @@ jobs:
        LEAK_CHECK=valgrind
 
 - job: linux_xenial_arm32
-  displayName: 'Linux (Xenial; arm32)'
+  displayName: 'Linux (Xenial; arm32; GCC)'
```

Comment:
Shouldn't these jobs then also say whether they use OpenSSL or mbedTLS, if we aim for consistency?